src/Nethermind/Nethermind.PerfTest/Program.cs

```diff
@@ -332,7 +332,7 @@ private static async Task RunBenchmarkBlocks()
                 if (allocation.Code != null)
                 {
                     Keccak codeHash = stateProvider.UpdateCode(allocation.Code);
-                    stateProvider.UpdateCodeHash(address, codeHash, specProvider.GenesisSpec);
+                    stateProvider.InsertCode(address, codeHash, specProvider.GenesisSpec);
```

Comment:
Isn't the second param the code itself?

Reply:
ahh, yes - my bad

Reply:
which solution is this a part of? i cant find the file